### PR TITLE
MF-64 - Fix db user role for postgresql-users

### LINKED ISSUE #64: Fix user role for postgresql-users
PostgreSQL DB user named `mainflux` by default is admin user. That is not enough privilege for latest users service becuase it creates extension in [init script](https://github.com/mainflux/mainflux/blob/3273c30d8b70d5808c69d9541b99b8fc27eaf3ea/users/postgres/init.go#L68) and for that we need  `superuser` role.
Bitnami PostreSQL chart that is used as dependency allow only `postgres` user to be superuser

### PR DESCRIPTION
**Pull request by blokovi**
Change postgresql-users user to postgres
Add new envars
Resolves #64 

Signed-off-by: Ivan Milosevic <iva@blokovi.com>

### CODE REVIEW

File: charts/mainflux/templates/users-deployment.yaml [hunk 1]
```diff
@@ -53,6 +53,10 @@ spec:
               value: {{ .Release.Name }}-envoy:{{ .Values.authn.grpcPort }}
             - name: MF_TOKEN_RESET_ENDPOINT
               value: "#/auth/reset-password"
+            - name: MF_USERS_ADMIN_EMAIL
```
Comment (drasko) [inline charts/mainflux/templates/users-deployment.yaml:56]: Admin of what? Some general admin, or only PotgreSQL admin?

**Review by drasko — CHANGES_REQUESTED** [review]

File: charts/mainflux/templates/users-deployment.yaml (hunk 1 above)
Comment (blokovi) [reply charts/mainflux/templates/users-deployment.yaml:56]: No database. It's Mainflux default user, created on startup

https://github.com/mainflux/mainflux/blob/master/users/README.md

Comment (drasko) [reply charts/mainflux/templates/users-deployment.yaml:56]: OK, I see. Thanks.

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**